net.danielgm.CloudCompare.yaml

```diff
@@ -0,0 +1,290 @@
+        sha256: 0add34d53cd27f8c468a59b8e931a636ad3174b60581c0387abb98a9fc9cddb6
+    buildsystem: cmake
+    builddir: true
+    config-opts:
+      - -DWITH_LIBUSB=OFF
+      - -DWITH_QT=OFF
+      - -DWITH_VTK=OFF
+      - -DWITH_PCAP=OFF
+      - -DPCL_ONLY_CORE_POINT_TYPES=ON
+      - -DOpenGL_GL_PREFERENCE=GLVND
+      - -DBUILD_2d=ON
+      - -DBUILD_CUDA=OFF
+      - -DBUILD_GPU=OFF
+      - -DBUILD_apps=OFF
+      - -DBUILD_common=ON
+      - -DBUILD_examples=OFF
+      - -DBUILD_features=ON
+      - -DBUILD_filters=ON
+      - -DBUILD_geometry=OFF
+      - -DBUILD_global_tests=OFF
+      - -DBUILD_io=ON
+      - -DBUILD_kdtree=ON
+      - -DBUILD_keypoints=ON
+      - -DBUILD_ml=OFF
+      - -DBUILD_octree=ON
+      - -DBUILD_outofcore=OFF
+      - -DBUILD_people=OFF
+      - -DBUILD_recognition=OFF
+      - -DBUILD_registration=OFF
+      - -DBUILD_sample_consensus=ON
+      - -DBUILD_search=ON
+      - -DBUILD_segmentation=OFF
+      - -DBUILD_simulation=OFF
+      - -DBUILD_stereo=OFF
+      - -DBUILD_surface=ON
+      - -DBUILD_tools=OFF
+      - -DBUILD_tracking=OFF
+      - -DBUILD_visualization=OFF
+
+  - name: CloudCompare
+    sources:
+      # use the source directory relative to manifest
+      - type: git
+        url: https://github.com/CloudCompare/CloudCompare.git
+        commit: 0c0fca2704218eb570af13a46aac2f1010796180
+      # patch: replace constexpr by const for floating point operations
+      # see https://github.com/CloudCompare/CloudCompare/issues/848
+      - type: patch
+        path: patches/cc_constexpr_double.patch
+    builddir: true
+    buildsystem: cmake
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=Release
+      - -DCOMPILE_CC_CORE_LIB_WITH_CGAL=ON
+      - -DCOMPILE_CC_CORE_LIB_WITH_TBB=ON
+      - -DOPTION_USE_GDAL=ON
+      - -DOPTION_USE_DXF_LIB=ON
+      - -DOPTION_USE_SHAPE_LIB=ON
+      - -DDLIB_ROOT=/app/include
+      - -DEIGEN_ROOT_DIR=/app/include/eigen3
+      - -DPLUGIN_GL_QEDL=ON
+      - -DPLUGIN_GL_QSSAO=ON
+      - -DPLUGIN_IO_QADDITIONAL=ON
+      - -DPLUGIN_IO_QCORE=ON
+      - -DPLUGIN_IO_QCSV_MATRIX=ON
+      - -DPLUGIN_IO_QE57=ON
+      - -DPLUGIN_IO_QFBX=OFF  # Autodesk FBX not available
+      - -DPLUGIN_IO_QPDAL=ON
+      - -DPLUGIN_IO_QPHOTOSCAN=ON
+      - -DPLUGIN_STANDARD_QANIMATION=ON
+      - -DPLUGIN_STANDARD_QBROOM=ON
+      - -DPLUGIN_STANDARD_QCANUPO=ON
+      - -DPLUGIN_STANDARD_QCOMPASS=ON
+      - -DPLUGIN_STANDARD_QCORK=OFF # error: no matching conversion for functional-style cast from 'const char [14]' to 'std::exception'
+      - -DPLUGIN_STANDARD_QCSF=ON
+      - -DPLUGIN_STANDARD_QFACETS=ON
+      - -DPLUGIN_STANDARD_QHOUGH_NORMALS=ON
+      - -DPLUGIN_STANDARD_QHPR=ON
+      - -DPLUGIN_STANDARD_QM3C2=ON
+      - -DPLUGIN_STANDARD_QPCL=ON
+      - -DPLUGIN_STANDARD_QPCV=OFF  # GL/glu.h: No such file or directory
+      - -DPLUGIN_STANDARD_QPOISSON_RECON=ON
+      - -DPLUGIN_STANDARD_QRANSAC_SD=ON
+      - -DPLUGIN_STANDARD_QSRA=ON
+
+  - name: launchers
```

Comment:
No need to have this as a separate module, you can just add them to CloudCompare & install them in post-install.